tools/packaging/packaging.psm1

```diff
@@ -1416,7 +1449,7 @@ function New-MacOSLauncher
     $plistcontent | Out-File -Force -Path $plist -Encoding utf8
 
     # Create shell script.
-    $executablepath = if ($IsPreview) { "/usr/local/bin/pwsh-preview" } else { "/usr/local/bin/pwsh" }
+    $executablepath = if ($IsPreview) { "/usr/local/bin/pwsh-preview" } elseif ($LTS) { "/usr/local/bin/pwsh-lts"} else { "/usr/local/bin/pwsh" }
```

Comment:
This is the 3rd place this code is duplicated

Reply:
refactored to a function